RobotCode/snobot2018/src/org/snobot/elevator/IElevator.java

```diff
@@ -0,0 +1,26 @@
+package org.snobot.elevator;
+
+/**
+ * This is the interface for the elevator that allows someone to move the
+ * elevator up and down.
+ * 
+ * @author Josh
+ *
+ */
+public interface IElevator
+{
+    /**
+     * Gets the elevator height in inches
+     * 
+     * @return the elevator height in inches
+     */
+    double getHeight();
+
+    /**
+     * Sets the motor speed when a button is pressed
+     * 
+     * @param aSpeed
+     */
+    void setMotorSpeed(double aSpeed);
```

Comment:
Do we really need an option to set the motor speed as a method? How often is that going to change?

Reply:
Ya why not